src/encoded/types/shared_calculated_properties.py

```diff
@@ -313,6 +313,8 @@ def biosample_summary(self,
                                 biosampleObject.get('post_synchronization_time_units'),
                                 biosampleObject.get('post_treatment_time'),
                                 biosampleObject.get('post_treatment_time_units'),
+                                biosampleObject.get('post_nucleic_acid_delivery_time'),
```

Comment:
Do you need to add a default value? Something like biosampleObject.get('post_nucleic_acid_delivery_time', '1')